website/docs/r/log_view.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "signalfx"
+page_title: "SignalFx: signalfx_log_view_chart"
+sidebar_current: "docs-signalfx-resource-log-view"
+description: |-
+  Allows Terraform to create and manage log views
+---
+
+# Resource: signalfx_log_view
+
+You can add logs data to your Observability Cloud dashboards without turning your logs into metrics first. A log view displays log lines in a table form in a dashboard and shows you in detail what is happening and why.
+
+## Example Usage
+
+```tf
+resource "signalfx_log_view" "mychart0" {
+  name        = "Sample Log View"
+  description = "Lorem ipsum dolor sit amet, laudem tibique iracundia at mea. Nam posse dolores ex, nec cu adhuc putent honestatis"
+
+  program_text = <<-EOF
+  logs(filter=field('message') == 'Transaction processed' and field('service.name') == 'paymentservice').publish()
+  EOF
+
+  time_range = 900
+  sort_options {
+    descending= false
+    field= "severity"
+   }
+
+  columns {
+        name="severity"
+    }
+  columns {
+        name="time"
+    }
+  columns {
+        name="amount.currency_code"
+    }
+  columns {
+        name="amount.nanos"
+    }
+  columns {
+        name="amount.units"
+    }
+  columns {
+        name="message"
+    }
+
+}
+```
+
+## Argument Reference
+
+The following arguments are supported in the resource block:
+
+* `name` - (Required) Name of the chart.
```

Comment:
Change all occurrences of `chart` to `log view`?

Reply:
done